### PR TITLE
fix license for 2019

### PR DESCRIPTION
**Pull request by jackjia-ibm**
Signed-off-by: Jack (T.) Jia <jack-tiefeng.jia@ibm.com>

### CODE REVIEW

File: data-sets-api-server/src/main/java/org/zowe/data/sets/DataSetsApplication.java [hunk 1]
```diff
@@ -1,12 +1,13 @@
-/*
+/**
  * This program and the accompanying materials are made available under the terms of the
  * Eclipse Public License v2.0 which accompanies this distribution, and is available at
  * https://www.eclipse.org/legal/epl-v20.html
  *
  * SPDX-License-Identifier: EPL-2.0
  *
- * Copyright IBM Corporation 2018
+ * Copyright IBM Corporation 2018, 2019
```
Comment (stevenhorsman) [inline data-sets-api-server/src/main/java/org/zowe/data/sets/DataSetsApplication.java]: This file hasn't updated this year

File: data-sets-api-server/src/main/java/org/zowe/data/sets/spring/SwaggerConfig.java [hunk 2]
```diff
@@ -1,12 +1,13 @@
-/*
+/**
  * This program and the accompanying materials are made available under the terms of the
  * Eclipse Public License v2.0 which accompanies this distribution, and is available at
  * https://www.eclipse.org/legal/epl-v20.html
  *
  * SPDX-License-Identifier: EPL-2.0
  *
- * Copyright IBM Corporation 2018
+ * Copyright IBM Corporation 2018, 2019
```
Comment (stevenhorsman) [inline data-sets-api-server/src/main/java/org/zowe/data/sets/spring/SwaggerConfig.java]: This file hasn't updated this year

File: data-sets-model/src/main/java/org/zowe/data/sets/model/AllocationUnitType.java [hunk 3]
```diff
@@ -1,12 +1,13 @@
-/*
+/**
  * This program and the accompanying materials are made available under the terms of the
  * Eclipse Public License v2.0 which accompanies this distribution, and is available at
  * https://www.eclipse.org/legal/epl-v20.html
  *
  * SPDX-License-Identifier: EPL-2.0
  *
- * Copyright IBM Corporation 2018
+ * Copyright IBM Corporation 2018, 2019
```
Comment (stevenhorsman) [inline data-sets-model/src/main/java/org/zowe/data/sets/model/AllocationUnitType.java]: This file hasn't updated this year

File: data-sets-api-server/src/main/java/org/zowe/data/sets/exceptions/UnauthorisedDataSetException.java [hunk 4]
```diff
@@ -1,12 +1,13 @@
-/*
+/**
  * This program and the accompanying materials are made available under the terms of the
  * Eclipse Public License v2.0 which accompanies this distribution, and is available at
  * https://www.eclipse.org/legal/epl-v20.html
  *
  * SPDX-License-Identifier: EPL-2.0
  *
- * Copyright IBM Corporation 2018
+ * Copyright IBM Corporation 2018, 2019
```
Comment (stevenhorsman) [inline data-sets-api-server/src/main/java/org/zowe/data/sets/exceptions/UnauthorisedDataSetException.java]: This file hasn't updated this year

**Comment by stevenhorsman** [comment]
I've always been told by IBM legal that files copyright dates should only be updated if there is "significant change", which I've been able to negotiate down to any change at all. Hence I've been manually updating every file I've touched in data-sets.

**Comment by jackjia-ibm** [comment]
I believe this is not necessary any more. Closing it.

**Pull request closed without merging**